README.md

```diff
@@ -5,9 +5,26 @@ Prezto — Instantly Awesome Zsh
 
 Prezto is the configuration framework for [Zsh][1]; it enriches the command line
 interface environment with sane defaults, aliases, functions, auto completion,
-and prompt themes. 
+and prompt themes.
 
-This repository is being actively maintained by a community of users. 
+Upgrading from sorin-ionescu/prezto
+-----------------------------------
+
+Prezto's [original maintainer][original] abandoned the project. This is a
+young, community-maintained fork, which was created [in December 2016][fork].
```

Comment:
I see the references below now, I'd just make them links here to make things easier, but it's okay.